Djmixpanel patch 1 (#1631)
mixpanel/docs: pages/docs/session-replay.mdx

| **CDP events** | Middleware required to watch replays that contain a specific event | You don’t need middleware to watch replays that contain specific events. However, we recommend adding middleware to handle edge cases where replays may not be associated with the right events (simultaneous sessions across tabs, domains, or apps) | **Medium** |
| **SDK-sent events** | Can easily watch replays that contain a specific event | Can easily watch replays that contain a specific event | **Low** |

In order for Server Side Stitching to work, just ensure that the Mixpanel Javascript SDK calls `identify()` at some point with the user's unique Distinct ID. [See managing user identity](/docs/tracking-methods/sdks/javascript#managing-user-identity).

## FAQ

### What happens when I reach my Session Replay limit?
For customers who have purchased our Session Replay add-on, you will be billed for replays ingested beyond your purchased amount at the per unit rate. Before this happens, we'll send email notifications warning that your limit is approaching to users with  “Owner” or “Billing Admin” permissions over your Mixpanel organization.

> For customers using Session Replay who have not purchased our add-on (for example, using complimentary replays included with your Free, Growth, or Enterprise plan), replays above your complimentary amount will not be viewable until the customer pays for overage replays (Enterprise plan only), or upgrades plan. Before this happens, users with  “Owner” or “Billing Admin” permissions over your Mixpanel organization will be notified that their limit is approaching.

Reviewer: ```suggestion
For customers using Session Replay who have not purchased our add-on (for example, using complimentary replays included with your Free, Growth, or Enterprise plan), replays above your complimentary amount will not be viewable until you upgrade your plan. Before this happens, users with  “Owner” or “Billing Admin” permissions over your Mixpanel organization will be notified that their limit is approaching.
```
